sdk/cosmos/azure-cosmos/test/test_vector_policy.py

```diff
@@ -0,0 +1,252 @@
+                    "distanceFunction": "euclidean"
+                }]}
+        indexing_policy = {
+            "vectorIndexes": [
+                {"path": "/vector1", "type": "flat"}]
+        }
+        container_id = "vector_container" + str(uuid.uuid4())
+        created_container = self.created_database.create_container(
+            id=container_id,
+            partition_key=PartitionKey(path="/id"),
+            indexing_policy=indexing_policy,
+            vector_embedding_policy=vector_embedding_policy
+        )
+        new_indexing_policy = {
+            "vectorIndexes": [
+                {"path": "/vector1", "type": "quantizedFlat"}]
+        }
+        try:
+            self.created_database.replace_container(
+                created_container,
+                PartitionKey(path="/id"),
+                indexing_policy=new_indexing_policy)
+            pytest.fail("Container replace should have failed for indexing policy.")
+        except exceptions.CosmosHttpResponseError as e:
+            assert e.status_code == 400
+            assert "Vector Indexing Policy cannot be changed in Collection Replace" in e.http_error_message
+        self.created_database.delete_container(container_id)
+
+    def test_fail_create_vector_embedding_policy(self):
+        # Using invalid data type
+        vector_embedding_policy = {
+            "vectorEmbeddings": [
+                {
+                    "path": "/vector1",
+                    "dataType": "float33",
+                    "dimensions": 1000,
+                    "distanceFunction": "euclidean"
+                }]}
+        try:
+            self.created_database.create_container(
+                id='vector_container',
+                partition_key=PartitionKey(path="/id"),
+                vector_embedding_policy=vector_embedding_policy
+            )
+            pytest.fail("Container creation should have failed but succeeded.")
+        except exceptions.CosmosHttpResponseError as e:
+            assert e.status_code == 400
+            assert "Vector Embedding Policy has invalid DataType" in e.http_error_message
+
+        # Using too many dimensions
+        vector_embedding_policy = {
+            "vectorEmbeddings": [
+                {
+                    "path": "/vector1",
+                    "dataType": "float32",
+                    "dimensions": 2000,
+                    "distanceFunction": "euclidean"
+                }]}
+        try:
+            self.created_database.create_container(
+                id='vector_container',
+                partition_key=PartitionKey(path="/id"),
+                vector_embedding_policy=vector_embedding_policy
+            )
+            pytest.fail("Container creation should have failed but succeeded.")
+        except exceptions.CosmosHttpResponseError as e:
+            assert e.status_code == 400
+            assert "Vector Embedding Policy has Dimensions:2000 which is more than max:1536" in e.http_error_message
```

Comment:
lol, 1536, we need to add these in our python docs until we have proper public documentation on these.